semcore/mini-chart/src/component/score/Line.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import createComponent, { Component, Root, sstyled, ComponentType } from '@semcore/core';
+import { Box, Flex, BoxProps } from '@semcore/flex-box';
+import resolveColorEnhance from '@semcore/utils/lib/enhances/resolveColorEnhance';
+
+import style from './line.shadow.css';
+
+export type CommonScoreProps = {
```

Comment:
Why you put common score props into specific score component module?

Reply:
moved to `Score.d.ts`